mikeio1d/various.py

```diff
@@ -66,9 +67,14 @@ def make_list_if_not_iterable(obj) -> list:
     list
         List with one element if obj is not iterable, otherwise obj.
     """
+    if obj is None:
+        return []
+
     try:
         iter(obj)
     except TypeError:
```

Comment:
As far as I know using try/except blocks is very non-performant if exception is triggered. Do we plan to use it more frequently, or is this intended for one of calls to modify user specified input? Would be the solution 3. in the top answer given in  https://stackoverflow.com/questions/1952464/in-python-how-do-i-determine-if-an-object-is-iterable is of any value here?